content/200-concepts/100-components/02-prisma-client/053-client-extensions/120-query.mdx

```diff
@@ -129,6 +136,25 @@ const xprisma = prisma.$extends({
 })
 ```
 
+### Modify all Prisma Client operations
+
+Use the `$allOperations` method to modify all query methods present in Prisma Client. The `$allOperations` can be used on both model operations and raw queries.
+
+You can modify all methods as follows:
+
+```ts
```

Comment:
Should we have an example here what `model`, `operations`, `args` and `query` actually contain when you call it? Maybe an "example example" below?

Reply:
Ideally, yes.

I didn't have the time to play with it yet, but I'll do my best to come up with a concrete example.

We could take this a step further, review all examples for Prisma Client extensions docs, and update them to have more concrete example use cases. The current Prisma Client extensions docs read like Reference docs. 😔

Reply:
I've added a query logging example let me know if that works